test/common/config/delta_subscription_state_test.cc

```diff
@@ -189,7 +189,7 @@ TEST_F(DeltaSubscriptionStateTest, CumulativeUpdates) {
 }
 
 // Verifies that a sequence of good and bad responses from the server all get the appropriate
-// ACKs/NACKs from Envoy.
+// ACKs/`NACKs` from Envoy.
```

Comment:
I think if ACK is in the dictionary, NACK should also be. It's common enough to be easily understood.

Reply:
Reverted Nacks